src/overlays/actors/ovl_En_Syateki_Okuta/z_en_syateki_okuta.c

```diff
@@ -370,7 +370,7 @@ void EnSyatekiOkuta_UpdateCollision(EnSyatekiOkuta* this, PlayState* play) {
     }
 
     this->collider.dim.pos.x = this->actor.world.pos.x;
-    // jointTable->y is the y-translation of the skeleton root
+    // jointTable->y is the y-translation of the skeleton root, see `LIMB_INDEX_POS`
     this->collider.dim.pos.y = this->actor.world.pos.y + (this->skelAnime.jointTable->y * this->actor.scale.y);
```

Comment:
Does `jointTable[0].y` not match? :o

Reply:
Nope, I messed around with it and it doesn't want to play nice :(

Reply:
... and `jointTable[0].y` also doesn't match in `z_en_okuta.c` in OoT :(